python/lancedb/table.py

```diff
@@ -292,7 +292,9 @@ def delete(self, where: str):
         --------
         >>> import lancedb
         >>> import pandas as pd
-        >>> data = pd.DataFrame({"x": [1, 2, 3], "vector": [[1, 2], [3, 4], [5, 6]]})
+        >>> data = pd.DataFrame([
```

Comment:
for delete and create, let's make the examples use list of dict instead of pandas dataframe

Reply:
Changed.